exercises/complex-numbers/src/example.c

```diff
@@ -0,0 +1,70 @@
+#include "complex_numbers.h"
+#include <math.h>
+
+double square(double x) {
+    return x * x;
+}
```

Comment:
Since this function is only used locally, can you declare it as `static` so that it is private to this module?

Reply:
You bet.